src/client/utils/tracking/tracking.ts

```diff
@@ -44,6 +44,14 @@ export const getUtmParamsFromCookie = (): UtmParams | undefined => {
   }
 }
 
+export enum SeBundleTypes {
+  SeHomeAccidentBundleStudentBrf = 'SE_ACCIDENT_BUNDLE_STUDENT_BRF',
+  SeHomeAccidentBundleBrf = 'SE_ACCIDENT_BUNDLE_BRF',
+  SeHomeAccidentBundleStudentRent = 'SE_ACCIDENT_BUNDLE_STUDENT_RENT',
+  SeHomeAccidentBundleRent = 'SE_ACCIDENT_BUNDLE_RENT',
+  SeHomeAccidentBundleHouse = 'SE_ACCIDENT_BUNDLE_HOUSE',
+}
```

Comment:
I created this type in my PR as well https://github.com/HedvigInsurance/web-onboarding/pull/655, but I think we can go with yours since you will need to know about rent/brf